media-source/mediasource-liveseekable.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html>
```

Comment:
style nit: no <html> <head> or <body> is what I was recently told is the new latest and greatest wpt style. Also, left-margin of 0 for the outermost layer of script body.